HOMEWORK2.HTML

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <h2>JavaScript For Loop</h2>
+  <p id="max_number"></p>
+  <p id="min_number"></p>
+  <p id="total"></p>
+  <script>
+    const numbers = [8, 1, 2, 3, 5, 6];
+    function findmax ()
+    {
+       let a = numbers.length;
+       let max = numbers [0];
+       for (let i=0; i<a; i++)
```

Comment:
you should start from index `1`, not `0` because we already get the `0` one as the max in the first place, it doesn't make sense if we compare numbers[0] to itself, does it?